episodes/02-setup.md

```diff
@@ -26,15 +26,15 @@ of configurations we will set as we get started with Git:
 - and that we want to use these settings globally (i.e. for every project).
 
 On a command line, Git commands are written as `git verb options`,
-where `verb` is what we actually want to do and `options` is additional optional information which may be needed for the `verb`. So here is how
-Alfredo sets up his new laptop:
+where `verb` is what we actually want to do and `options` is additional optional information which may be needed for the `verb`. To set up a new computer:
 
 ```bash
-$ git config --global user.name "Alfredo Linguini"
-$ git config --global user.email "a.linguini@ratatouille.fr"
+$ git config --global user.name "Joanne Simpson"
+$ git config --global user.email "j.simpson@weather.com"
```

Comment:
Probably best to go for a made up domain!

Reply:
mo-weather.uk might be a bit more future proof - I would expect quite a few people to copy paste whatever you put without altering it. The approach in my old team - altered since someone complained, so can't recommend it! - was to set it as e.g.:

```
git config --global user.email "user-does-not@follow-instructions.com"  # Please alter this to your own email
```

Reply:
Ok final change to `mo-weather.uk` :)